### PR TITLE
[SRS] Add FRs for employee management

### PR DESCRIPTION
**Pull request by LeonSo7**
[SRS]
- Add employee management subsection
- Add FRs for employee invitation
- Add FRs for viewing employees
- Add FRs for searching employees
- Add FRs for removing employees

Depends on [PR#95](https://github.com/arkinmodi/project-sayyara/pull/95) for BE numbering.

### CODE REVIEW

File: docs/SRS/SRS.tex
Comment (joyxiao99) [inline docs/SRS/SRS.tex:331]: 'invite an employee'

**Comment by LeonSo7** [comment]
Do not merge until #95 is merged.

File: docs/SRS/SRS.tex
Comment (LeonSo7) [reply docs/SRS/SRS.tex]: Fixed, thanks for catching that.

**Review by arkinmodi — CHANGES_REQUESTED** [review]

File: docs/SRS/SRS.tex
Comment (LeonSo7) [reply docs/SRS/SRS.tex]: Updated

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**
